packages/core-ethereum/src/indexer/index.ts

```diff
@@ -919,21 +945,33 @@ class Indexer extends (EventEmitter as new () => IndexerEventEmitter) {
   }
 
   private async onChannelClosed(channel: ChannelEntry) {
-    await this.db.deleteAcknowledgedTicketsFromChannel(channel)
+    await this.db.delete_acknowledged_tickets_from(Ethereum_ChannelEntry.deserialize(channel.serialize()))
     this.emit('channel-closed', channel)
   }
 
   private async onEligibilityUpdated(
     event: RegistryEvent<'EligibilityUpdated'>,
     lastSnapshot: Snapshot
   ): Promise<void> {
+    assert(lastSnapshot !== undefined)
```

Comment:
This will crash the node, is that the desired behaviour? Can it recover upon restart from it?

Reply:
See above.